src/leo-client-app/src/components/Logs.tsx

```diff
@@ -18,14 +18,22 @@ import "./styles/logs.css";
 import "../styles.css";
 import moment from "moment";
 import { BACKEND_URL } from "@/constants/api";
+import { useRouter } from "next/router";
 
 type Props = {
   satelliteId: string;
 };
 
 const Logs: React.FC = () => {
-  // TODO: Dynamicall get satelliteId from somewhere
-  const satelliteId = "655acd63d122507055d3d2ea";
+  const router = useRouter();
+  let { satId } = router.query as {
+    satId: string;
+  };
+  if (!satId) {
+    satId = "655acd63d122507055d3d2ea";
+  }
+
+  const satelliteId = satId;
```

Comment:
similar to above, but instead

const satId = router.query.satId?.toString() ?? "655acd63d122507055d3d2ea";